src/data-visualisation/power-bi/learning/certifications/AI-exams/ai-index.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: collection
+title: "Microsoft Azure certifications (AI)"
+description: "Overview of the Microsoft Azure certifications relevant to the NHSBSA"
+tags: pbi-learning
+order: 30
+collection_tag: ai-certs
+pagination:
+  data: collections.ai-certs
+  size: 50
+---
+There are many Microsoft Power Platforms and Azure certifications available. Please look at the pages available or the [attached spreadsheet][link] for more details.  
+  
+If you have any questions please get in touch with [Helen O'Donnell (Data Visualisation Manager)][email address].
+
+[link]: https://nhsbsauk.sharepoint.com/:x:/s/DataWarehouse/EccsayGlP-JDqSFfWK5BpXIBp3Ghr3wxMrjHUOtHmKQfCw?e=l1fvBH
```

Comment:
Internal link only accessible by those within nhs bsa?

Reply:
Forgot about this one. Will have to discuss alternatives with HOD next week.